frontend/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "frontend",
   "homepage": "https://ngutn24.github.io/RollBuddy",
+  "proxy": "http://localhost:4567",
```

Comment:
Should this be backend's port number? 4567 is backend, 3030 is frontend

Reply:
yes it's meant to proxy to the api endpoint!